azext_edge/edge/params.py

```diff
@@ -302,6 +310,822 @@ def load_iotops_arguments(self, _):
             options_list=["--name", "-n"],
             help="Dataflow endpoint name.",
         )
+        context.argument(
+            "authentication_type",
+            options_list=["--auth-type"],
+            choices=CaseInsensitiveList(
+                [
+                    DataflowEndpointAuthenticationType.ACCESSTOKEN.value,
+                    DataflowEndpointAuthenticationType.SASL.value,
+                    DataflowEndpointAuthenticationType.SERVICEACCESSTOKEN.value,
+                    DataflowEndpointAuthenticationType.SYSTEMASSIGNED.value,
+                    DataflowEndpointAuthenticationType.USERASSIGNED.value,
+                    DataflowEndpointAuthenticationType.X509.value,
+                ]
+            ),
+            help="The authentication type for the dataflow endpoint. "
+            "Note: When not specified, the authentication type is determinded by other authentication parameters.",
+        )
+        context.argument(
+            "client_id",
+            options_list=["--client-id", "--cid"],
+            help="The client ID of the user assigned identity",
+            arg_group="User Assigned Managed Identity",
+        )
+        context.argument(
+            "scope",
+            options_list=["--scope"],
+            help="Resource identifier (application ID URI) of the resource, affixed with the .default suffix.",
+            arg_group="User Assigned Managed Identity",
+        )
+        context.argument(
+            "tenant_id",
+            options_list=["--tenant-id", "--tid"],
+            help="The tenant ID of the user assigned identity.",
+            arg_group="User Assigned Managed Identity",
+        )
+        context.argument(
+            "no_auth",
+            options_list=["--no-auth"],
+            arg_type=get_three_state_flag(),
+            help="No authentication for the endpoint.",
```

Comment:
Not sure if disable auth would be better here

Reply:
I think `--no-auth` is short and sweet. It succinctly conveys the config. What is wrong with it? `--disable-auth` works, but I'm not a fan of `--anonymous-auth`  or `--anon-auth` for a flag that indicates no auth config will be used for the endpoint.